css-size-adjust-1/Overview.bs

```diff
@@ -97,54 +108,90 @@ The default size adjustment affects text and form controls, whether
 those form controls contain text (e.g., text inputs, selects) or do not
 (e.g., radio buttons, checkboxes).
 
-Conditions that suppress adjustment {#default-adjustment-conditions}
---------------------------------------------------------------------
+Adjustment Conditions {#default-adjustment-conditions}
+------------------------------------------------------
+
+A User Agent may consider any information when determining which elements
+get adjusted.
 
-A number of conditions suppress the default adjustment because these
-conditions are associated with layouts for which the user experience
-would be worsened by size adjustment rather than improved by it.  These
-conditions are:
+Note: In general, User Agents are trying to determine which elements are
+integral to the structure of the page, and thus shouldn't be adjusted,
+and which elements are allowed to flow freely and thus can be adjusted
+without undesirable rendering artifacts.
 
+Here are some criteria which User Agents may consider when making this
+determination:
```

Comment:
I think this list is still all worded in the form of things that would or might lead to a decision *not* to do text size adjustment.  So I think it's still worth saying that, so it's clear.  (The previous header for the list says that -- and you also said it explicitly inside the first item, but I think it's true for all the items.  And if it's not, i.e., if some of the items are the other way around, then I think it's worth restructuring them to match the rest.)